packages/emotion/src/EmotionThemeProvider/index.tsx

```diff
@@ -79,16 +104,15 @@ function EmotionThemeProvider({ children, theme = {} }: any) {
  * If an override object is given, it returns the ancestor theme and
  * the overrides merged together.
  *
- * @param {object} theme - A full theme or an override object
+ * @param {object} themeOrOverride - A full theme or an override object
  * @returns {function} A function that returns with the theme object for the [ThemeProvider](https://emotion.sh/docs/theming#themeprovider-reactcomponenttype)
+ * @module getTheme
  */
-const getTheme = (theme: any) => (ancestorTheme = {}) => {
-  // themeable themes have a 'key' property (= name of the theme),
-  // so without it it's just an overrides objects
-  const overrides = !theme.key ? theme : null
-
-  if (!overrides) {
-    return theme
+const getTheme = (themeOrOverride: ThemeOrOverride) => (
+  ancestorTheme: BaseTheme = canvas
```

Comment:
I did it this way because `{}` is not assignable to the `BaseTheme` type and since we are already default to `canvas` in the `useTheme` hook, I thought this will be fine to do.

Reply:
Well, not sure it is a widely used use-case, but if someone wanted to use the EmotionThemeProvider on the top level, and wrap their app in their own InstUI compatible theme, it will always merge on top of canvas then, and might cause some unwanted results.

Reply:
You are right, changed back to be `{}` and did a type casting instead